File: VDS_Frontend/src/app/acceso-temporal/Models/generar-acceso-request.dto.ts
export class GenerarAccesoRequestDTO {
    usuarioId: number;
    mascotaId: number;
  
    constructor(usuarioId: number, mascotaId: number) {
      this.usuarioId = usuarioId;
      this.mascotaId = mascotaId;
    }
}
  

File: VDS_Backend/src/main/java/com/uoc/tfm/vds_backend/acceso_temporal/dto/GenerarAccesoRequestDTO.java
package com.uoc.tfm.vds_backend.acceso_temporal.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class GenerarAccesoRequestDTO {
    private Long usuarioId;
    private Long mascotaId;
}
